src/wp-includes/block-editor.php

```diff
@@ -361,6 +361,89 @@ function _wp_get_iframed_editor_assets() {
 	);
 }
 
+/**
+ * Finds the first occurrence of a specific block in an array of blocks.
+ *
+ * @since 6.3.0
+ *
+ * @param string $block_name Name of the block to find.
+ * @param array  $blocks     Array of blocks.
+ * @return array Found block, or empty array if none found.
+ */
+function wp_get_first_block( $block_name, $blocks ) {
```

Comment:
This may be just a personal preference, but I would recommend to reverse the order of those parameters. Here's why:
* If `$blocks` had some default value and was optional, it would make sense that it's the 2nd parameter. But given that it has to be provided, I think it's more intuitive to put it first because that's the source data where to look in.
* It's intuitive that for such a function you surely have to pass the block name / identifier, but whether or not you have to pass the source to look in is less obvious (e.g. in a function like `get_post()` you don't have to provide a source), which to me is another reason to clarify that by putting that source parameter first.
* Last but not least, it's also a bit closer in the order to common code patterns like `someArray.find( value )`, basically "blocks -> find -> block_name".

Reply:
Sounds good!